app/models/appointment.rb

```diff
@@ -23,7 +23,7 @@ class Appointment < ApplicationRecord
     visited: "visited"
   }, _prefix: true
 
-  enum cancel_reason: CallResult.remove_reasons
+  enum cancel_reason: CallResult.remove_reasons.except(:already_visited)
```

Comment:
This is a new `remove_reason` that wasn't previously supported. The `cancel_reason` enum used to be defined inline here, but we moved it to `CallResult.remove_reasons` in a [previous PR](https://github.com/simpledotorg/simple-server/pull/2861/files)